sdks/java/core/src/main/java/org/apache/beam/sdk/values/PCollection.java

```diff
@@ -330,6 +330,24 @@ public Schema getSchema() {
     return ((SchemaCoder) getCoder()).getSchema();
   }
 
+  /** Returns the attached schema's toRowFunction. */
+  @Experimental(Kind.SCHEMAS)
+  public SerializableFunction<T, Row> getToRowFunction() {
```

Comment:
I think this doesn't belong on `PCollection`, they're not a property of it. I'd rather have something like `T getCoder(Class<T>)` overload to help with casting